lib/puppet/type/user.rb

```diff
@@ -400,10 +400,9 @@ def insync?(current)
     end
 
     newproperty(:expiry, :required_features => :manages_expiry) do
-      desc "The expiry date for this user. Must be provided in
-           a zero-padded YYYY-MM-DD format --- e.g. 2010-02-19.
-           If you want to ensure the user account never expires,
-           you can pass the special value `absent`."
+      desc "The expiry date for this user. Provide as either the special
+           value `absent` to ensure that the account never expires,
+           or as a zero-padded YYYY-MM-DD format --- e.g. 2010-02-19.
```

Comment:
Since we're updating this anyway, it would be nice for it to conform to our style guide and use "for example, 2010-02-19" instead of "e.g." which isn't very localization friendly.